src/Sqs/SqsClient.php

```diff
@@ -124,14 +190,47 @@ private function validateMd5()
                     ->then(
                         function ($result) use ($c, $r) {
                             foreach ((array) $result['Messages'] as $msg) {
+                                $bodyMd5 = self::calculateBodyMd5($msg);
```

Comment:
This was because the implicit else condition of `!isset($msg['MD5OfMessageAttributes'])` is pre-confirmed on the initial if, when we check to see `isset($msg['MessageAttributes'])`.